src/components/course/data/hooks.jsx

```diff
@@ -282,6 +282,7 @@ export const useCourseEnrollmentUrl = ({
   sku,
   userSubsidyApplicableToCourse,
   isExecutiveEducation2UCourse,
+  isAlreadyEnrolled,
```

Comment:
nit: Would it make sense to name this `isUserEnrolled` to be consistent with the naming elsewhere?